src/components/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,29 @@
+import { Link } from 'react-router-dom';
+import React from 'react';
+import classNames from 'classnames';
+import { Person } from '../../types/Person';
+
+interface Props {
+  person: Person;
+}
+
+export const PersonLink: React.FC<Props> = ({ person }) => {
+  const {
+    name,
+    sex,
+    slug,
+  } = person;
+
+  const isWoman = sex === 'f';
+
+  return (
+    <Link
+      to={`../${slug}`}
+      className={classNames({
```

Comment:
Change the 'to' property value to include the correct path for the person's page. Use `to={`/people/${slug}`}` instead of `to={`../${slug}`}`.